portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,136 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let map;
+const locationsDict = new Object();
+
+class Location {
+  constructor(marker, lat, lng, header, pitch, infoString, infoWindow) {
+    this.marker = marker;
+    this.lat = lat;
+    this.lng = lng;
+    this.header = header;
+    this.pitch = pitch;
+    this.infoString = infoString;
+    this.infoWindow = infoWindow;
+  }
+}
+
+/** Calls all necessary functions to be called onLoad */
+function initialize() {
+  loadComments();
+  createMap();
+}
+
+/** Creates a map with all components and adds it to the page. */
+function createMap() {
+  map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 41.8996099, lng: -87.6232902, h: -7, p: 22 },
+    zoom: 13,
+  });
+
+  setupMarkers();
+  changeView(locationsDict["Bean"]);
```

Comment:
Since this call is tightly coupled with the previous call (you have to call setupMarkers first for the "Bean" location to be registered), I think I would actually just put this call in setupMarkers() itself:
```
function setupMarkers() {
 ...
 changeView(locationsDict["Bean"]);
}`
```
If you did this you could also clean up other references to locationsDict and make locationsDict a local variable of setupMarkers.